azerothcore/azerothcore-wotlk
# fix(DB/Quest): A Portabale Power Source

## CHANGES PROPOSED:
-  Change location of the POI

## TESTS PERFORMED:
- Builds
- tested in-game



## HOW TO TEST THE CHANGES:

Current 
![image](https://user-images.githubusercontent.com/24550914/96174405-8c7cd880-0f29-11eb-9a31-3a3a260acddc.png)

How it should be (Since the item drops from Magma Lord Bokk that is in Burning Steppes
![image](https://user-images.githubusercontent.com/24550914/96174422-94d51380-0f29-11eb-99cb-6c7801876190.png)

1. .q add 8925
2. Open map and see that the Quest POI is in the wrong zone

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
